src/main/webapp/i18n/de/result.json

```
@@ -85,7 +87,10 @@
             "notLatestSubmission": "Hinweis: Dieses Ergebnis und Feedback bezieht sich <strong>nicht</strong> auf die letzte Einreichung.",
             "seeMore": "Mehr anzeigen",
             "noFeedback": "Kein Feedback verfügbar",
-            "afterDueDateFeedbackHidden": "Auch nach der Abgabefrist werden Detailfeedbacks nicht zu allen Testfällen angezeigt. Dies ist etwa der Fall wenn für andere Studierende eine spätere Abgabefrist gesetzt ist (z.B. zum Ausgleich von Krankheitstagen). Alle Feedbacks werden automatisch am {{date}} sichtbar sein."
+            "afterDueDateFeedbackHidden": "Auch nach der Abgabefrist werden Detailfeedbacks nicht zu allen Testfällen angezeigt. Dies ist etwa der Fall wenn für andere Studierende eine spätere Abgabefrist gesetzt ist (z.B. zum Ausgleich von Krankheitstagen). Alle Feedbacks werden automatisch am {{date}} sichtbar sein.",
+            "testCount": "Du hast {{passed}} von {{total}} Tests bestanden",
+            "scaIssueCount": "Du hast {{issues}} Code-Issues",
+            "manualFeedbackCount": "Du hast {{feedbacks}} manuelle Feedbacks bekommen"
```

```suggestion
            "testCount": "Du hast {{passed}} von {{total}} Tests bestanden.",
            "scaIssueCount": "Du hast {{issues}} Code-Issues.",
            "manualFeedbackCount": "Du hast {{feedbacks}} manuelle Feedbacks bekommen."
```
I think those should get a full stop at the end as they are proper sentences.